docs/multicluster/getting-started.md

```diff
@@ -0,0 +1,429 @@
+# Antrea Multi-cluster Installation
+
+Antrea Multi-cluster works on several Kubernetes clusters to allow user to export and import
```

Comment:
Antrea Multi-cluster allows users to export and import resources including Services and Endpoints across multiple clusters within a Cluster, and enables inter-cluster Service communication in the ClusterSet.

Reply:
done